src/stratis_cli/_actions/_physical.py

```diff
@@ -39,8 +52,35 @@ def list_pool(namespace):
         """
         List devices in a pool.
         """
-        # pylint: disable=unused-argument
-        raise StratisCliUnimplementedError("No ability to list pools.")
+        proxy = get_object(TOP_OBJECT)
+        managed_objects = ObjectManager.Methods.GetManagedObjects(proxy, {})
+        (parent_pool_object_path, _) = pools(
+           managed_objects,
+           props={'Name': namespace.pool_name},
+           unique=True
+        )
+        modevs = [MODev(info) for _, info in devs(
+            managed_objects,
+            props={"Pool": parent_pool_object_path},
+        )]
+        tables = [
+            [
+                modev.Devnode(),
+                modev.TotalPhysicalSize(),
+                state_num_to_string(modev.State()),
```

Comment:
Better to print out something disturbing if state numeric code really is unexpected, something like:
```UNEXPECTED NUMBER: 32```